clang/docs/checkedc/3C/README.md

```diff
@@ -48,3 +50,66 @@ that 3C will contain the core inference logic, while 5C will add
 features to enhance developer productivity. If you'd like more
 information about 5C, please contact us at
 info@correctcomputation.com.
+
+### Which `checkedc-clang` repository to use?
+
+We typically recommend that serious 3C users use CCI's repository to
+get 3C fixes and enhancements sooner, but in some scenarios, you may
+be better off with Microsoft's repository. Here, in detail, are the
+factors we can think of that might affect your decision (as of
+February 2021):
```

Comment:
March 2021 ?

Reply:
Well, I wrote that in February 2021, but I agree the situation is not going to change in the next 2 days. :slightly_smiling_face: Done.